run-debug

```diff
@@ -1,13 +1,4 @@
 #!/bin/bash
 
-# Simple shell script to run debug mode. If using 6.2 or below use `--debug` otherwise use `--inspect`
-
-CURRENT_NPM_VERSION=$(node -v)
-CURRENT_NPM_VERSION=$(node -e "console.log(parseFloat(\"$CURRENT_NPM_VERSION\".replace(\"v\", \" \")).toFixed(1))")
```

Comment:
FWIW this line had a bug if, for example, Node.js 6.10.x-6.15.x was used as the code would end up printing "6.1" which is not "> 6.2" in the line below.